README.md

```diff
@@ -49,6 +51,26 @@ the image first.
 
 - When confirmed, click on the `Flash` button to start flashing procedure.
 
+### bmaptool
+
+If you prefer a command-line tool, then you should use bmaptool, just install it with
```

Comment:
@WiktorG351  Let's mention here that it is a way to install bmap-tools on Ubuntu or other distro using apt package manager. Also, please use more formal tone when writing documentation. So maybe do not use `just install it with...` use smth like `To install it, please run the following command.

Then we can provide two options, one for distros with apt and the other for distros with dnf